client/apps/game/src/ui/features/world/containers/mini-map-navigation.tsx

```
@@ -1,7 +1,9 @@
 import { ReactComponent as CollapseIcon } from "@/assets/icons/common/collapse.svg";
 import { ReactComponent as ExpandIcon } from "@/assets/icons/common/expand.svg";
 import { useUIStore } from "@/hooks/store/use-ui-store";
+import CircleButton from "@/ui/elements/circle-button";
 import { ResourcesIds } from "@bibliothecadao/types";
+import { Map } from "lucide-react";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)